### PR TITLE
Fix my speaker bio

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@debian.org>

### CODE REVIEW

File: content/2018-munich/speakers/richard-hartmann.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ title: Richard Hartmann
 
 ## Richard Hartmann
 
-Richard is a Prometheus team member, currently focusing on building a new datacenter for SpaceNet, Richard comes from the networking side of things and has been on call 24/7 for over a decade in his last life. He learned the hard way that precise monitoring which supports operations is key to keeping people sane, and allowing them both sleep and a social life.
+Richard is a Prometheus team member, currently focusing on building a new datacenter for SpaceNet. He comes from the networking side of things and has been on call 24/7 for over a decade in his last life. He learned the hard way that precise monitoring which supports operations is key to keeping people sane, and allowing them both sleep and a social life.
```
Comment (brian-brazil) [inline content/2018-munich/speakers/richard-hartmann.md]: past life?

Comment (RichiH) [reply content/2018-munich/speakers/richard-hartmann.md]: It's the one directly before this life; but I am ambivalent.

Comment (brian-brazil) [reply content/2018-munich/speakers/richard-hartmann.md]: Last life would imply reincarnation usually.

**Comment by brian-brazil** [comment]
:+1:

File: content/2018-munich/speakers/richard-hartmann.md (hunk 1 above)
Comment (RichiH) [reply content/2018-munich/speakers/richard-hartmann.md]: That's pretty much the point, yes.

**Pull request merged**